tests/Feature/TestAPI.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use PHPUnit\Framework\TestCase;
+
+class TestAPI extends TestCase
+{
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function testOngs()
+    {
+        $response = $this->get('/api/ongs');
+
+        // $response->assertStatus(200);
+        $response->assertJson(
+            [
+              "id"=> 0,
+              "name"=> "APAD",
+              "email"=> "contato@apad.com.br",
+              "whatsapp"=> "64000000000",
+              "city"=> "Rio do Sul",
+              "uf"=> "SC"
+            ],
+            [
+              "id"=> 66714519,
+              "name"=> "Galileu",
+              "email"=> "galileu@contatogalileu.com.br",
+              "whatsapp"=> "6400000000",
+              "city"=> "Catalão",
+              "uf"=> "GO"
+            ]);
```

Comment:
A validade do seu teste está dependendo do ambiente. Se eu não possuir esses dados no BD de teste, irá falhar...

Lembrando que um feature test realiza o teste da aplicação e não dos dados existentes no BD.

Se você estiver testando somente a estrutura de retorno do endpoint, recomendo o `assertJsonStructure`.

Lembrando que no feature test você teria que inserir algo antes no banco.

O ideal é que uma feature test recrie os dados no BD a cada testCase. [Laravel - Resetando banco de dados a cada teste](https://laravel.com/docs/7.x/database-testing#resetting-the-database-after-each-test)